src/Dhall/Parser.hs

```diff
@@ -446,6 +447,7 @@ exprF embedded = choice
             ,   noted      exprF05
             ,   noted      exprF06
             ,   noted      exprF07
+            ,   noted      exprF35
```

Comment:
this position was chosen rather arbitrarily, can you give some guidance what makes sense here?